docs/_sass/_search-bar.scss

```diff
@@ -0,0 +1,197 @@
+@import 'breakpoints';
+@import 'colors';
+@import 'fonts';
+
+$color-appBG: $color-green-appBG;
+$color-highlightBG: $color-green-highlightBG;
+$color-accent : $color-green400;
+$color-borders: $color-green-borders;
+$color-icons: $color-green-icons;
+$color-text: $color-white;
+$color-link: $color-blue300;
+$color-link-hovered: $color-blue200;
+$color-success: $color-green400;
+$color-text-supporting: $color-light-gray-green;
+$color-green-hover: $color-green-hover;
+$color-gray-label: $color-gray-label;
```

Comment:
Ah why are we reassigning the color variables to new variables? The code would be more maintainable if we user the original variables directly

Reply:
This is the pattern we have on the codebase @marcochavezf. If we refactor it, would it be another scope then?

Reply:
From what I see, why it's re-assigned is because some of the variables are color names 
e.g.
```
$color-text: $color-white;
$color-link: $color-blue300;
$color-link-hovered: $color-blue200;
$color-success: $color-green400;
```